2d/platformer/src/Main/Game.tscn

```diff
@@ -14,3 +14,28 @@ script = ExtResource( 2 )
 layer = 100
 
 [node name="PauseMenu" parent="InterfaceLayer" instance=ExtResource( 1 )]
+
+[node name="GameOverScreen" type="Control" parent="InterfaceLayer"]
+visible = false
+margin_right = 800.0
+margin_bottom = 480.0
+__meta__ = {
+"_edit_use_anchors_": false
+}
+
+[node name="ColorRect" type="ColorRect" parent="InterfaceLayer/GameOverScreen"]
+margin_right = 800.0
+margin_bottom = 480.0
+color = Color( 0.0392157, 0.0392157, 0.0392157, 1 )
+
+[node name="GameOverLabel" type="Label" parent="InterfaceLayer/GameOverScreen"]
+margin_left = 282.0
+margin_top = 206.0
+margin_right = 519.0
+margin_bottom = 254.0
+text = "You are not good enough....
+
+I suggest getting better at the game.."
```

Comment:
These responses are not insulting enough. I suggest adding phrases that will slowly wear at the psyche of player, driving them into madness.

Reply:
lol. Good call. I'll add a few more and randomly display one upon game over